test/integration/local/test_dummy.py

```diff
@@ -50,3 +50,34 @@ def test_install_requirements(capsys):
     assert "Installing collected packages: pyfiglet" in stdout
     assert "Successfully installed pyfiglet-0.8.post1" in stdout
     assert "Reporting training SUCCESS" in stdout
+
+
+# def test_install_requirements_from_codeartifact(capsys):
```

Comment:
Will you enable this test later?

Reply:
So, for this to really be enabled, it requires that the build infrastructure for this package to actually have a codeartifact repo which I believe it doesn't.
I've written the integ test, that can be enabled if the build infra has such access.

So, at the moment, the answer is no, I don't think I can enable this on my own.